node/pkg/driver/node_utils.go

```diff
@@ -84,10 +85,15 @@ func (n NodeUtils) GetInfoFromPublishContext(publishContext map[string]string, c
 	}
 
 	connectivityType := publishContext[configYaml.Controller.Publish_context_connectivity_parameter]
-	array_iqns := strings.Split(publishContext[configYaml.Controller.Publish_context_array_iqn], ",")
+	if connectivityType == "iscsi" {
+		array_initiators = strings.Split(publishContext[configYaml.Controller.Publish_context_array_iqn], ",")
```

Comment:
Please don't name variables in this way: "array_initiators" !!!
golang use CamelCase naming convention

Reply:
Done